koku/api/common/permissions/settings_access.py

```diff
@@ -11,6 +12,32 @@ class SettingsAccessPermission(permissions.BasePermission):
 
     resource_type = "settings"
 
+    def has_permission(self, request, view):
+        """Check permission based on the defined access."""
+        if settings.ENHANCED_ORG_ADMIN and request.user.admin:
+            return True
+
+        if not request.user.access:
+            return False
+
+        if request.method in permissions.SAFE_METHODS:
+            setting_read = request.user.access.get(self.resource_type, {}).get("read", [])
+            if setting_read:
```

Comment:
nit: an assignment expression could be used here.
```suggestion
            if setting_read := request.user.access.get(self.resource_type, {}).get("read", []):
```

Reply:
Since we are just returning true, I don't think we need the assignment actually. 

```
if request.user.access.get(self.resource_type, {}).get("read", []):
```
Should work just find for us.

Reply:
Absolutely. This is much better.